src/MergeContext.js

```diff
@@ -1243,11 +1242,21 @@ class PullRequest {
 
     /* _ex*() methods below are mutually exclusive: first match wins */
 
+    // instant retries does not require reprocessing from scratch
+    _exRetryNow(why) {
+        assert(arguments.length === 1);
+        this._reprocessingDelayMs = 0;
```

Comment:
Delay value aside, _exRetryNow() looks just like _exRetrySoon(), but they are supposed to have very different effect. See the `Push-Labels` column values in the exceptions summary table. AFAICT, that column is correct (and has different values for these two exceptions).

Reply:
Agreed, I forgot to set _labelPushBan. Done.